src/mgr/PyModuleRegistry.h

```diff
@@ -40,6 +40,7 @@ class PyModuleRegistry
 {
 private:
   mutable Mutex lock{"PyModuleRegistry::lock"};
+  mutable Mutex config_lock{"PyModuleRegistry::config_lock"};
```

Comment:
A reference to this lock will be needed by anyone who has a reference to `module_config` (e.g. StandbyPymodules and soon ActivePyModules), it's probably easiest to do that by bundling the Mutex and std::map together into a little class.